data/human/hails.txt

```diff
@@ -1708,6 +1708,78 @@ phrase "friendly civilian"
 		", but I still had to take a big loan to buy my first ship."
 		", but it was my grandma's inheritance that allowed me to buy this ship."
 
+phrase "friendly civilian"
+	word
+		"Don't be too picky about the jobs you take. It is better to"
+	word
+		" "
+	word
+		" ship shit, than having a shitty ship."
```

Comment:
I must admit that I don't like this one. It is alliterative, and definitely flows off the tongue. I just don't like the language. While it should be admittedly rare to have it come up, I still wouldn't want kids playing a game with this language.

```suggestion
```

Reply:
Good feedback. Thanks!